File: app/src/main/java/vn/loitp/app/activity/demo/nfc/TagWrapper.java
package vn.loitp.app.activity.demo.nfc;

class TagWrapper {
    private String id;
    TagTechList techList = new TagTechList();

    public TagWrapper(final String tagId) {
        id = tagId;
    }

    public final String getId() {
        return id;
    }
}

File: core/src/main/java/com/loitp/core/utilities/nfc/TagWrapper.kt
package com.loitp.core.utilities.nfc

import androidx.annotation.Keep
import com.loitp.core.base.BaseModel

/**
 * Created by Loitp on 04,August,2022
 * Galaxy One company,
 * Vietnam
 * +840766040293
 * freuss47@gmail.com
 */
@Keep
class TagWrapper(val id: String) : BaseModel() {
    var techList = TagTechList()
}
